source/_cookbook/send_a_reminder.markdown

```diff
@@ -29,7 +29,7 @@ and automation part to your `configuration.yaml` file.
 automation:
   - alias: Send message at a given time
     trigger:
-      platform: time
+      platform: time_pattern
```

Comment:
Good catch, but that whole automation should be changed. For using a specific time, we should use 
```yaml
    platform: time
    at: '12:15:00'
```

Reply:
Yes, this makes more sense.